test/integration/collection-management/collection.test.ts

```diff
@@ -95,18 +95,14 @@ describe('Collection', function () {
       ]);
     });
 
-    it('should fail due to illegal listCollections', function (done) {
-      expect(() => db.collection(5)).to.throw('Collection name must be a String');
-      expect(() => db.collection('')).to.throw('Collection names cannot be empty');
-      expect(() => db.collection('te$t')).to.throw("Collection names must not contain '$'");
-      expect(() => db.collection('.test')).to.throw(
-        "Collection names must not start or end with '.'"
-      );
-      expect(() => db.collection('test.')).to.throw(
-        "Collection names must not start or end with '.'"
-      );
-      expect(() => db.collection('test..t')).to.throw('Collection names cannot be empty');
-      done();
+    it('should fail on server due to illegal collection name', async function () {
```

Comment:
`Collection standard collection tests should fail on server due to illegal collection name`

If I am following the indentation of the tests correctly, the above is the full title of this test. I think it is important to mention the specific API under test and the intended behavior to understand what our goal was if we need to revisit the decisions made here. Also, nit, we can drop the "it ( should xx" for imperative "it ( fails xx"

Reply:
Okay, sounds good